kolibri/logger/csv.py

```diff
@@ -35,11 +35,13 @@ def get_channel_name(self, obj):
         return channel.name
 
     def get_content_title(self, obj):
-        channel = ChannelMetadata.objects.get(id=obj.channel_id)
-        node = ContentNode.objects.filter(tree_id=channel.root.tree_id).first()
-        if node:
-            return node.title
-        else:
+        try:
+            node = ContentNode.objects.filter(content_id=obj.content_id).first()
+            if node:
+                return node.title
+            else:
+                return ""
+        except ContentNode.DoesNotExist:
```

Comment:
isn't this exception redundant given the above code. on an empty queryset `first()` returns `None`, so when would this exception get hit?

Reply:
True, true!